### DISCUSSION TITLE
Web App's Loading time

### DISCUSSION THREAD

**Discussion by dakshverma22** [alagrede/znote-app#14] [General]
The app itself is useful but the loading time for the web app is too long.
Maybe due to the images, I don't know much about that, but the loading time is way too long.

**Comment by alagrede** [comment]
Thank you for your report. That's right, the site is clearly not well optimized ... I hope it wouldn't bother browsing too much until I fix it.

**Comment by dakshverma22** [comment]
Happy to help 😊
